### PR TITLE
Bump to version 0.5.2, new custom filter version that relays 429s

### PR DESCRIPTION
**Pull request by tpboudreau**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # 0.5.1
```
Comment (itzg) [inline CHANGELOG.md]: Cosmetic, but looks like you missed the version bump in the changelog

**Comment by jjbuchan** [comment]
Nice, I didn't see you'd already done https://github.com/rackerlabs/cloud-monitoring-filter-bundle/pull/12

+1

In case you've forgotten, we'll also have to bump the git tag and then do a chef pr once this gets merged.

**Comment by rphillips** [comment]
Looks like a lint error: https://travis-ci.org/mmi-cookbooks/ele-repose/jobs/173973393#L592

**Comment by tpboudreau** [comment]
@rphillips, yes, but those files are unchanged...

**Comment by dimtruck** [comment]
We should ignore that error...it's basically saying that the code block should be 25 lines...but it's in Vagrant and our, very easily read, default spec test.

@mburns thoughts?

**Comment by rphillips** [comment]
+1

**Pull request merged**